src/holdingpen/components/ExceptionsDashboard.jsx

```diff
@@ -48,15 +48,10 @@ class ExceptionsDashboard extends Component {
             )}
           />
         </div>
-
-        <Row type="flex" justify="space-around">
-          <Col span={20} justify="center" align="middle">
```

Comment:
why did you move those to the `ExceptionsTable`, couldn't you just make this `Col` as `<Col span={20}>`?